src/ott/solvers/outputs.py

```diff
@@ -0,0 +1,66 @@
+from typing import Optional, Tuple
+
+import jax.numpy as jnp
+from flax import struct
+
+from ott.geometry import geometry
+from ott.problems.linear import linear_problem
+
+
+@struct.dataclass
+class _BaseTransportOutput:
+  """Base class for Transport output.
+
+  Args:
+    ot_prob: OT problem.
+    errors: Holds sequence of vectors of errors of the Sinkhorn algorithm
```

Comment:
This should not refer to Sinkhorn at all, it should just state this is the array used to determine the convergence of the algorithm.